doc/classes/GPUParticles2D.xml

```diff
@@ -92,5 +106,21 @@
 		</constant>
 		<constant name="DRAW_ORDER_REVERSE_LIFETIME" value="2" enum="DrawOrder">
 		</constant>
+		<constant name="EMIT_FLAG_POSITION" value="1" enum="EmitFlags">
+			Particle starts at the specified position.
+		</constant>
+		<constant name="EMIT_FLAG_ROTATION_SCALE" value="2" enum="EmitFlags">
+			Particle starts with specified rotation and scale.
+		</constant>
+		<constant name="EMIT_FLAG_VELOCITY" value="4" enum="EmitFlags">
+			Particle starts with specified velocity.
+			Keep in mind that this refers also to the emission direction
```

Comment:
Missing dot on the second sentence, but assuming I understood the intended description properly, rephrasing like this might be clearer:
```suggestion
			Particle starts with the specified velocity vector, which defines the emission direction and speed.
```